File: sqlInsert.py
import MySQLdb

db = MySQLdb.connect(host="localhost", user="user", passwd="password", db="school")
cur = db.cursor(MySQLdb.cursors.DictCursor)

db.autocommit(True)

#take user input
name = input("Enter your  name ")
age = input("Enter your age ")
gradeLevel = input("Enter your grade level ")

#insert user input into the database
sql = (f"INSERT INTO students (name,age,gradeLevel) VALUES ('{name}',{age},{gradeLevel})")
cur.execute(sql)

File: sqlInsert.php
<?php
$servername = "localhost";
$username = "user";
$password = "password";
$dbname = "school";
$name = 'Test';
$age = 15;
$gradeLevel = 11;
// Create connection
$conn = new mysqli($servername, $username, $password, $dbname);
// Check connection
if ($conn->connect_error) {
    die("Connection failed: " . $conn->connect_error);
}

$sql = "INSERT INTO students(name,age,gradeLevel) VALUES('$name','$age','$gradeLevel') ";
$result = $conn->query($sql);

if ($conn->query($sql) === TRUE) {
    echo "New record created successfully";
} else {
    echo "Error: " . $sql . "<br>" . $conn->error;
}
$conn->close();
?> 
